Scripts/BlackwingLair: fix Nefarian Stage 3 (Bone Construct respawn)

14shagov opened the pull request:
<!--- (**********************************)
      (** Fill in the following fields **)
      (**********************************) --->

**Changes proposed:**

-  The corpses of mobs killed during phase 1 are revived as bone constructs in phase 3

**Issues addressed:**

Closes https://github.com/TrinityCore/TrinityCore/issues/23046


**Tests performed:**

build, tested in-game


<!--- Notes
- Enable the setting "[√] Allow edits from maintainers." when creating your pull request.
- If this PR only contains SQL files, open a new issue instead and post or link the SQL in the issue.
- When adding new SQL files, name them 9999_99_99_99_db_name.sql to reduce the chance of possible merge conflicts.
--->

Commit:
Nefarian / fix respawn

Commit:
fix

Commit:
fix

CraftedRO commented:
Just tested this PR and it looks a lot better than what we have, maybe will merge it this century.

CraftedRO commented:
bump

CraftedRO commented:
@14shagov can you also add this https://github.com/TrinityCore/TrinityCore/issues/23046#issuecomment-706394046 / https://youtu.be/JNGTL8ZA6-o?t=494 ?

14shagov committed:
Merge branch '3.3.5' into Nefarian_respawn

14shagov committed:
fix

- Added SAY_XHEALTH and SAY_SHADOWFLAME.
- Changed 'creature_text' so that SAY_AGGRO, SAY_XHEALTH and SAY_SHADOWFLAME are in different groups.
